snapshot/bundlestore/server.go

```diff
@@ -43,11 +43,12 @@ func MakeServer(s store.Store, ttl *store.TTLConfig, stat stats.StatsReceiver, g
 // Implements http.Handler interface
 func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	s.storeConfig.Stat.Counter(stats.BundlestoreRequestCounter).Inc(1)
+	//TODO(apratti): write ttl in headers in each of these
```

Comment:
I don't think this is part of the contract this server's API guarantees

Reply:
Will remove